11-docker-compose/13-pingdataconsole-pingone-sso/docker-compose.yaml

```diff
@@ -0,0 +1,78 @@
+version: "2.4"
+#-------------------------------------------------------------------------------------
+#- Ping Identity PingDataConsole SSO with PingOne demo
+#-
+#- The PingDataConsole can be configured to use SSO with PingOne. Once you've configured an
+#- application in PingOne for the console, you can replace the PD_CONSOLE_SSO_ISSUER_URI,
+#- PD_CONSOLE_SSO_CLIENT_ID, and PD_CONSOLE_SSO_CLIENT_SECRET variables in this file with
+#- the corresponding values from PingOne. You will also need to create a user in
+#- PingOne that corresponds to a root user DN configured in PingDirectory. The expected root
+#- user in this demo is Jane Smith (username jsmith).
+#-
+#-                          +---------------+
+#-                          |PingDataConsole|
+#-                          +---------------+
+#-                                  |
+#-                          +----------------+
+#-                          | PingDirectory  |
+#-                          +----------------+
+#-
+#- To view PingData Console goto:
+#- 
+#-    https://localhost:8443/
+#-    
+#-      Server: pingdirectory
+#-    Username: administrator
```

Comment:
If SSO is enabled for the console, it won't be possible to log in through regular username or password anymore -- the console will always try to redirect to the OIDC server.

Reply:
I've removed these values and replaced them with a sentence describing that you will be redirected and that you'll login with a PingOne user.